io.posidon.Paper.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "io.posidon.Paper",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
```

Comment:
```suggestion
    "runtime-version" : "41",
```

Reply:
Can't do it, cuz the app needs libadwaita, I guess I'll just wait for 42

Reply:
you can also build libadwaita like other packages have done.